CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Improved
 - The `@tinymce/no-main-module-imports` rule now detects Main imports via aliases from outside the `src/main/` directory.
-- The `@tinymce/prefer-fun` rule now detects where `Fun.constant` should be used.
+- The `@tinymce/prefer-fun` rule now detects where `Fun.constant` and `Fun.identity` should be used.
```

Comment:
I was going to ask why you're changing the changelog entries for a release that's apparently already gone out.

But apparently 1.9.0 hasn't gone out (the latest release was 1.8.3, according to npm, which isn't even in the changelog).

Should we be moving (or at least shifting the date for) that 1.9.0 heading?

Reply:
Yeah, we will need to. I was expecting faster reviews tbh :shrug: